packages/larva/scripts/build-scss.js

```diff
@@ -0,0 +1,15 @@
+const path = require( 'path' );
```

Comment:
Note: the contents of the Gulp command files are largely the same - it's fine for now.